distros/kubernetes/nvsentinel/charts/mongodb-store/Chart.lock

```diff
@@ -2,5 +2,11 @@ dependencies:
 - name: mongodb
   repository: ""
   version: 16.2.2
-digest: sha256:7bc0222d2ca507e5b3c38829401a888800575ea76a693b42b4ab8edb2cd0ff37
-generated: "2025-09-19T18:55:22.424746878+05:30"
+- name: psmdb-operator
+  repository: https://percona.github.io/percona-helm-charts
```

Comment:
As a final commit before we merge this PR, can we vendor in the chart as well like we have done for bitnami?